docs/source/guide/start.md

```diff
@@ -91,6 +93,24 @@ nginx:
 
 To run Label Studio on Docker with a host and subdomain, you can refer to the example `deploy/dockerfiles/subpath.example.yml` Docker YAML file. To customize it for your environment, manually modify the `nginx/subdomain.example.simple.conf` NGINX configuration file, the relevant Label Studio environment variables, and the PostgreSQL database settings for your environment. 
 
+## Run Label Studio on Docker and use local storage
+To run Label Studio on Docker and reference persistent local storage directories, mount those directories as volumes when you start Label Studio and specify any environment variables you need.
+
+The following command starts a Docker container with the latest image of Label Studio with port 8080 and an environment variable that allows Label Studio to access local files. It also mounts a source directory to use for data files, in this example, a Docker volume at `/var/lib/docker/volumes/ls-data-files`, and a target directory to persist Label Studio data such as annotations, at `/label-studio/data-store` on the machine running Docker.
+```bash
+docker run -it -p 8080:8080 --env LABEL_STUDIO_LOCAL_FILES_SERVING_ENABLED=true,LABEL_STUDIO_LOCAL_FILES_DOCUMENT_ROOT="/var/lib/docker/volumes/" --mount type=volume,source=ls-task-files,target=/label-studio/data-store heartexlabs/label-studio:latest label-studio
+```
+Label Studio assumes the root of the local file directory path is `/`. To specify a different file directory root, use the `LABEL_STUDIO_LOCAL_FILES_DOCUMENT_ROOT` environment variable. In this example, the root is updated to specify the path to where Docker stores volumes. 
+
+You can also reference a local file directory of `/etc/apps/label-studio/task-files` with the source argument like this example, assuming you are in the `opt` directory when you run the following command:
+```bash
+docker run -it -p 8080:8080 --env LABEL_STUDIO_LOCAL_FILES_SERVING_ENABLED=true --mount type=bind,source="$(pwd)/etc/apps/label-studio/task-files",target=/label-studio/data-store heartexlabs/label-studio:latest label-studio
```

Comment:
@niklub a bind mount is required to access the local file system from Docker. I've tested this out on my local machine (referencing a folder on my desktop) but haven't been able to get the e2e flow working successfully. Would appreciate it if you test this out as well.

Reply:
The right command is
```
docker run -it -p 8080:8080 -v `pwd`/mydata:/label-studio/data \
 --env LABEL_STUDIO_LOCAL_FILES_SERVING_ENABLED=true --env LABEL_STUDIO_LOCAL_FILES_DOCUMENT_ROOT=/label-studio/files -v `pwd`/myfiles:/label-studio/files \
heartexlabs/label-studio:latest label-studio
```

so it basically mounts the local `./myfiles` directory from the host machine to the internal `/label-studio/files`. And then `LABEL_STUDIO_LOCAL_FILES_DOCUMENT_ROOT=/label-studio/files` tells to scan only this directory

Reply:
Also worth mentioning about that it's highly recommended to always secure docker container by explicit LABEL_STUDIO_LOCAL_FILES_DOCUMENT_ROOT